recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.08.14" %}
+{% set version = "2018.08.21" %}
```

Comment:
Do you think a build number bump is more appropriate?

Reply:
Not certain @conda-forge/core ?

Reply:
New version is correct